articles/getting-started-with-matlab/README.md

```diff
@@ -0,0 +1,207 @@
+# Getting-started-with-MATLAB
+### Introduction
+MATLAB (an abbreviation of "matrix laboratory") is a programming language and numerical computing environment developed by MathWorks.
+
+### Prerequisites
+- Good understanding of any programming language.
+- MATLAB installed, you can download it from [here](https://www.mathworks.com/downloads/).
+
+### Uses of MATLAB
+- Signal Processing and Communications.
+- Data analysis and visualization.
+- Scientific and engineering graphics.
+- Developing algorithms.
+- Modeling, simulation, and prototyping.
+
+MATLAB has an additional package "Simulink" which you can use for graphical multi-domain simulation and model-based design for dynamic and embedded systems. And you can get access to the symbolic computing abilities by using an optional toolbox that uses the MuPAD symbolic engine.
+
+### Lets code!
+In heart, MATLAB is a calculator. For example, try: `1+1`, `sqrt(4)`, or `sin(pi/2)`.
```

Comment:
At the core of MATLAB lies efficient matrix computations. It is not a calculator.

Reply:
These commands can be through the interpreter.